src/gpu_common.c

```diff
@@ -126,7 +126,9 @@ void nvidia_probe(void)
 	if (nvml_lib)
 		return;
 
-	if (!(nvml_lib = dlopen("libnvidia-ml.so", RTLD_LAZY|RTLD_GLOBAL)))
+	if (!(nvml_lib = dlopen("libnvidia-ml.so", RTLD_LAZY|RTLD_GLOBAL)) &&
+	    !(nvml_lib = dlopen("libnvidia-ml.so.1", RTLD_LAZY|RTLD_GLOBAL)) &&
```

Comment:
I suggest `.so.1` first, fallback to `.so` next.

Reply:
Did so